### DISCUSSION TITLE
A space required at the beginning of trailing comments?

### DISCUSSION THREAD

**Discussion by hchiba1** [pg-format/pg-formatter#38] [Q&A] [answered]
Currently, a space is required at the beginning of trailing comments.

By disallowing `#` in unquoted characters, they would not require the space.

**Comment by nichtich** [comment, marked as answer]
`#` is allowed in URIs (for instance `http://www.w3.org/2004/02/skos/core#Concept`) so I'd prefer to allow it in unquoted values. Putting a comment right after a value without space is less readable and no good coding style anyway.

**Reply by hchiba1** [reply]
Reasonable. I think so, too.
